packages/mapviewer/tests/cypress/tests-e2e/featureSelection.cy.js

```diff
@@ -91,6 +91,8 @@ describe('Testing the feature selection', () => {
             goToMapViewWithFeatureSelection('TOoLtIp')
             checkFeatures()
             checkFeatureInfoPosition(FeatureInfoPositions.BOTTOMPANEL)
+            // verifying that the left table cell is not too close to the right table cell
+            cy.get('td.cell-left').should('have.css', 'padding-right', '10px')
```

Comment:
I'd really prefer we use some `data-cy` selector here ([see best practices for Cypress](https://docs.cypress.io/app/core-concepts/best-practices#Selecting-Elements))
I know it might be tricky, because this code comes from the backend. But we could add our `data-cy` directly in the fixture if needed